docs/examples/tutorial_HD_dataset.py

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Peyrache et al (2015) Dataset Tutorial
+============
+
+This tutorial demonstrates how we use Pynapple on various publicly available datasets in systems neuroscience to streamline analysis. In this tutorial, we will examine the dataset from [Peyrache et al (2015)](https://www.nature.com/articles/nn.3968), which was used to generate Figure 4a in the [publication](https://elifesciences.org/reviewed-preprints/85786).
+
+The NWB file for the example used here is provided in [this](https://github.com/PeyracheLab/pynacollada/tree/main/pynacollada/Pynapple%20Paper%20Figures/Peyrache%202015/Mouse32/Mouse32-140822/pynapplenwb) repository. The entire dataset can be downloaded [here](https://dandiarchive.org/dandiset/000056).
+
+See the [documentation](https://pynapple-org.github.io/pynapple/) of Pynapple for instructions on installing the package.
+
+This tutorial was made by Dhruv Mehrotra.
+
+First, import the necessary libraries:
+    
+"""
+# %%
+
+import numpy as np
+import pandas as pd
+import pynapple as nap
+import scipy.ndimage
+import matplotlib.pyplot as plt
+
+# %%
+# ***
+# Head-Direction Tuning Curves
+# ------------------
+#
+# The first step is to load the data and other relevant variables of interest
+
+data_directory = (
+    "/media/DataDhruv/Recordings/Mouse32/Mouse32-140822"  # Path to your data
+)
+
+data = nap.load_file("Mouse32-140822.nwb")  # Load the NWB file for this dataset
```

Comment:
Not sure this will work. Did you run the code?

Reply:
Fixed